doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -87,6 +87,11 @@ nRF Desktop
 
   * Added documentation for :ref:`nrf_desktop_usb_state_pm`.
 
+* Removed:
+
+  * Removed configuration files for building the application with SW_SPLIT LL for various boards.
```

Comment:
I might be wrong, but isn't this :kconfig:`SW_SPLIT_LL`?

Reply:
its :kconfig:`BT_LL_SW_SPLIT` it wasn't my intention to put the exact name but why not. I will push in a minute.